.github/workflows/python-app.yml

```diff
@@ -0,0 +1,62 @@
+# This workflow will install Python dependencies, run tests and lint with a single version of Python
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: Python application
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+env:
+  NODE_VERSION: '10.x'                # set this to the node version to use
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python 3.8
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.8
+        node-version: ${{ env.NODE_VERSION }}
+    - name: Install dependencies
+      run: |
+        python3 -m pip install --upgrade pip
+        if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
+    - name: Aspell install
+      run: |
+        sudo apt-get update -qq
```

Comment:
This is because of github action issue mentioned in https://github.com/actions/virtual-environments/issues/599
for Aspell it is also specified as the way to install and with out sudo it doesnt https://howtoinstall.co/en/ubuntu/trusty/aspell